spec/models/green_flag/feature_spec.rb

```diff
@@ -132,4 +132,53 @@
       end
     end
   end
+
+  describe '#requires_manual_deletion?' do
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.